hollow/src/main/java/com/netflix/hollow/tools/diff/HollowDiff.java

```diff
@@ -60,7 +64,7 @@
      * @param to the "to" state
      */
     public HollowDiff(HollowReadStateEngine from, HollowReadStateEngine to) {
-        this(from, to, true);
+        this(from, to, true, false);
```

Comment:
nit: passing a final static boolean for eg. `DONT_INCLUDE_NON_PRIMARY_KEY_TYPES = false` might be good for readability

Reply:
Fair point but I was just going with the flow since existing code base does not do that :)

Also IDEs like Intellij seems to do a good job showing the param names.